src/screen/Parachains/ParachainDetailScreen.tsx

```diff
@@ -1,102 +1,231 @@
+import {useParachainValidators} from 'src/api/hooks/useParachainValidators';
+import type {
+  AccountId,
+  CoreAssignment,
+  ParaValidatorIndex,
+  ParaId,
+  CandidatePendingAvailability,
+} from '@polkadot/types/interfaces';
+import {EmptyView} from 'presentational/EmptyView';
+import IdentityIcon from '@polkadot/reactnative-identicon/Identicon';
+import {useAccountIdentityInfo} from 'src/api/hooks/useAccountIdentityInfo';
+import AccountInfoInlineTeaser from 'presentational/AccountInfoInlineTeaser';
+import {useParachainInfo} from 'src/api/hooks/useParachainInfo';
+import {useParaEndpoints} from 'src/api/hooks/useParaEndpoints';
 
 type ScreenProps = {
   route: RouteProp<ParachainsStackParamList, 'parachainDetail'>;
 };
 
+type ParachainValidators = {
+  validators: AccountId[];
+  assignments?: CoreAssignment[];
+  validatorGroups: ParaValidatorIndex[][];
+  validatorIndices?: ParaValidatorIndex[];
+};
+
+function getValidatorInfo(id: string, parachainValidators?: ParachainValidators) {
+  const assignment = parachainValidators?.assignments?.find(({paraId}) => paraId.eq(id));
+
+  if (!assignment) {
+    return undefined;
+  }
+
+  return {
+    groupIndex: assignment.groupIdx,
+    validators: parachainValidators?.validatorGroups[assignment.groupIdx.toNumber()]
+      ?.map((indexActive) => [indexActive, parachainValidators?.validatorIndices?.[indexActive.toNumber()]])
+      .filter(([, a]) => a)
+      .map(([, indexValidator]) =>
+        indexValidator ? parachainValidators.validators[indexValidator?.toNumber()] : undefined,
+      ),
+  };
+}
+
+function getNonVoters(validators?: AccountId[], pendingAvail?: CandidatePendingAvailability) {
+  let list: AccountId[] = [];
+
+  if (validators && pendingAvail) {
+    list = pendingAvail.availabilityVotes
+      .toHuman()
+      .slice(2)
+      .replace(/_/g, '')
+      .split('')
+      .map((c, index) => (c === '0' ? validators[index] : null))
+      .filter((v, index): v is AccountId => !!v && index < validators.length);
+  }
+
+  return list;
+}
+
 export function ParachainDetailScreen({route}: ScreenProps) {
   const {id, name, period, blocks} = route.params;
   const theme = useTheme();
 
   const events = useParachainEvents();
+  const {data: parachainValidators} = useParachainValidators();
+  const {data: parachainInfo} = useParachainInfo(id as unknown as ParaId);
+
+  const validatorInfo = useMemo(() => getValidatorInfo(id, parachainValidators), [id, parachainValidators]);
+  const nonVoters = useMemo(
+    () => getNonVoters(parachainValidators?.validators, parachainInfo?.pendingAvail),
+    [parachainValidators, parachainInfo],
+  );
+
+  const endpoints = useParaEndpoints(id as unknown as ParaId);
+  const homepage = endpoints?.length ? endpoints[0]?.homepage : undefined;
+
+  const sections = [
+    {
+      title: `Val. Group ${validatorInfo?.groupIndex.toNumber() || ''} (${validatorInfo?.validators?.length || 0})`,
+      data: validatorInfo?.validators || [],
```

Comment:
notEmpty is a function in utils if you would like to get rid of the `undefined`s in the array
```suggestion
      data: validatorInfo?.validators.filter(notEmpty) || [],
```